src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -964,6 +971,144 @@ public static String composeRegularCallableMethodHeaderSampleCode(
     return bodyStatements;
   }
 
+  private static List<Statement> composePagedCallableSampleCodeBodyStatements(
+      Method method,
+      VariableExpr clientVarExpr,
+      VariableExpr requestVarExpr,
+      Map<String, Message> messageTypes) {
+    List<Statement> whileBodyStatements = new ArrayList<>();
```

Comment:
Can we move this to the line just before it is first used?

Reply:
Sure. Will do.